resources/views/Item/SingleItemWrapper.twig

```diff
@@ -68,16 +68,30 @@
 {% endif %}
                     "url"           : "{{ requestedVariationUrl }}",
 {% if itemData.prices.default.baseLot > 0 %}
-                    "priceSpecification": {
-                        "@type": "UnitPriceSpecification",
-                        "price": "{{ itemData.prices.default.price.value | number_format(2, '.', '') }}",
-                        "priceCurrency": "{{ item.documents[0].data.prices.default.currency }}",
-                        "referenceQuantity": {
-                            "@type": "QuantitativeValue",
-                            "value": "{{ itemData.unit.content }}",
-                            "unitCode": "{{ itemData.unit.unitOfMeasurement }}"
+                    "priceSpecification":[
+                        {
+                            "@type": "UnitPriceSpecification",
+                            "price": "{{ itemData.prices.default.price.value | number_format(2, '.', '') }}",
+                            "priceCurrency": "{{ item.documents[0].data.prices.default.currency }}",
+                            "priceType": "SalePrice",
+                            "referenceQuantity": {
+                                "@type": "QuantitativeValue",
+                                "value": "{{ itemData.unit.content }}",
+                                "unitCode": "{{ itemData.unit.unitOfMeasurement }}"
+                            }
+                        },
+                        {
+                            "@type": "UnitPriceSpecification",
+                            "price": "{{ itemData.prices.rrp.price.value | number_format(2, '.', '') }}",
```

Comment:
rrp is not required and a lot of items has no rrp, this should lead to an error .

Reply:
check if one rrp exists